cypress/support/step-definitions/textbox-steps.js

```diff
@@ -229,3 +229,20 @@ Then('Multiple label Align on preview is {string}', (direction) => {
   labelByPosition(FIRST_ELEMENT).should($element => expect($element).to.have.css(TEXT_ALIGN, `${direction}`));
   labelByPosition(SECOND_ELEMENT).should($element => expect($element).to.have.css(TEXT_ALIGN, `${direction}`));
 });
+
+Then('I click on icon inside of Textbox', () => {
+  icon().click();
+});
+
+Then('icon on preview is {string} and is visible', (iconName) => {
+  icon().should('have.attr', 'data-element', iconName)
+    .and('be.visible');
+});
+
+When('I click on Textbox', () => {
+  textbox().click();
+});
+
+Then('Textbox input has golden border on focus', () => {
+  textbox().should('have.css', 'outline', 'rgb(255, 181, 0) solid 3px');
```

Comment:
we need to verify cy.focused() element, not just the textbox

Reply:
I can't use cy.focused() because I click on the input, and the parent element has a golden border